editor/components/block-styles/style.scss

```diff
@@ -2,27 +2,40 @@
 	display: grid;
 	grid-template-columns: 1fr 1fr;
 	grid-template-rows: 60px;
-	grid-gap: 5px;
+	grid-gap: $item-spacing;
 }
 
 .editor-block-styles__item {
 	cursor: pointer;
-	border: 1px solid $light-gray-500;
+	outline: 1px solid transparent; // Shown in Windows High Contrast mode.
+	box-shadow: inset 0 0 0 1px rgba( $dark-gray-900, .2 );
 	overflow: hidden;
-	padding: 5px;
+	padding: 0;
 	text-align: initial;
+	border-radius: $button-style__radius-roundrect;
+
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    flex-direction: column;
 
 	> * {
 		transform: scale( 0.7 );
 		transform-origin: center center;
 		font-family: $editor-font;
 	}
 
+	// Make vertical centering easier.
+	> blockquote.wp-block-quote,
```

Comment:
I think this is too specific and probably not something we want to fix in the `block-styles` stylesheet.

Reply:
Open to suggestions, but the issue this fixes is that it enables true vertical centering of the contents inside the preview, by making sure the margins don't offset the balance.

Reply:
I think we need to figure out another way for this. I got it almost working with `align-self: center`.

Reply:
Well, if we added yet another wrapping container element, then we could apply some large margins above and below that element. Through the magic of collapsing margins, the largest margin would win, and because it would be the same above and below, then centering would work.

Reply:
Disabling the keyboard requires a wrapper, I'm adding a wrapper for the preview not sure if this would fix the issue though.